exercises/tom_clayton/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,158 @@
+/**
+ * Import statements
+ */
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <algorithm>
+#include <cctype>
+#include <string.h>
+#include <vector>
+#include <map>
+using namespace std;
```

Comment:
Not sure if this is documented but we don't use "using namespace std". Although it's makes the code more verbose by having the std:: in a lot of places, it means there's not any danger of name clashes between names in the Mantid code and the standard library

Reply:
Thanks, removed and made sure std:: is inserted where needed